### ISSUE TITLE
Create a webpage for impress.js

### ISSUE THREAD

**Issue by fsdias** [impress/impress.js#645]
I think you should create a webpage for the project to advertise new features and new releases. Demos are useful to showcase the software's features and capabilities but are not a good way to communicate new releases, share tutorials or templates.

I'm really happy that relative positioning, substeps and markdown support were added. I think these features greatly improve the software's usability. IMHO creating a webpage would be a good way to create momentum for the project.

Thanks a lot for your work.  I'm really happy the project is moving forward after such a prolonged hiatus and I look forward to testing all this new stuff.

**Comment by simplePCBuilding** [comment]
I agree and I am working on something currently. I am not that great at marketing, but I think I can pull something off

**Comment by simplePCBuilding** [comment]
Done, opened a PR
